octopusCats/src/main/scala/octopus/cats/package.scala

```diff
@@ -0,0 +1,23 @@
+package octopus
+
+import _root_.cats.data.{Validated, ValidatedNel, NonEmptyList}
+
+package object cats {
+
+  implicit class OctopusCatsOps[T](val vr: ValidationResult[T]) extends AnyVal {
+
+    def toValidatedNel: ValidatedNel[ValidationError, T] = vr.errors match {
+      case Nil =>
+        Validated.Valid(vr.value)
+      case head :: tail =>
+        Validated.Invalid(NonEmptyList.of(head, tail : _*))
```

Comment:
Can be as well, both are equivalent. But for me current version seems to be shorter and more readable.